EMCAL/EMCALbase/AliEMCALTriggerSTUDCSConfig.h

```diff
@@ -86,9 +86,6 @@ class AliEMCALTriggerSTUDCSConfig : public TObject
   void    GetSegmentation(TVector2& v1, TVector2& v2, TVector2& v3, TVector2& v4) const;
   
   TClonesArray *GetErrorCountsForTRU(Int_t itru) const;
-  
-protected:
```

Comment:
I think the removal of the protected is the cause of the failure, you should add it at least for the = operator that you have not created, but I am not sure, you could try it